content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,60 @@ tags:
 
 ## How Climb Works
 
-Climb works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.
+### Strafing
+
+Strafing is the most common focus for maps and what sets Climb apart from the Bhop and surf game modes. Strafing can come in the form of long jumps and taking indirect lines.
+**Airstrafing** works in the same way that bunny hopping does; using a strafe key (A or D) while guiding the mouse off in that direction causes the player to receive more speed. A **prestrafe** is a tehcnique that has been patch out of counterstrike since CSGO but has been re-added to this gamemode. While on the ground, players hold **both** the W and strafe keys and guide the mouse into the direction they are moving, right for W+D and left for W+A. Perfoming a prestrafe correctly while let you gain speed on the ground. Gaining speed before the jump and while in the air lets you travel futher. This is called a **long jump**. There are 2 other forms of longjump in 1.6 that are more advanced, [countjumps](https://kz-rush.com/en/page/countjump-physics) and [highjumps](https://kz-rush.com/page/highjump-physics).<br>
+<br>
+Note that since strafing on ground and in the air causes the player to deviate from their intended destination (assuming it is straight in front of them), prestrafing is usually done only before jumps rather than constantly, and long jumps typically incoroporate multiple left and right airstrafes into a jump which counteract eachother and keep the player close to a straight line.<br><br>
+
+Sometimes you may not be able to take a direct line due to an obstruction. The 2 most common forms of this is rounding a corner (often without being able to see the next platform) and 2 platforms separated by a thin wall.
+In this case, air strafing is not used to gain speed but as a form of air control. The principle is the same but typically smooth turns at the cost of speed are needed (don't take out all your speed or you'll fall straight down!).<br>
+
+### Bhop
+
+Bhopping has reduced potential compared to the Bhop game mode. Bhopping has its own speedcap: **(?) Momkz** &emsp;**380 KZT** &emsp; **300 1.6 Climb**.<br>
+This cap is seperate from the **maxvelocity** cap (how fast the engine lets you go before not bothering to give anymore speed 3500 by default).<br>  
+Anytime you jump the Bhop speed cap will be used meaning even if the player gained speed purely through falling or surfing, as soon as they try to jump off the ground with this speed, the cap will proc.
+If the player reaches this cap and tries to jump again, thier speed will be reset to:<br>
+**? (MomClimb)** &emsp; **? (KZT)** &emsp; **240 (1.6 Climb)**.<br>
```

Comment:
KZT resets you to 380 speed, momkz will probably stay around the 300 mark, which is to be determined in the future.